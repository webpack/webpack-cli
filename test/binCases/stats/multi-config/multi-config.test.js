"use strict";

const { run } = require("../../../testUtils");

test("multi-config", () => {
	const { code, stdout, stderr } = run(__dirname);

	expect(code).toBe(0);
	expect(stdout).toEqual(expect.anything());
	expect(stderr).toHaveLength(0);
	expect(stdout).toMatchSnapshot();
});
