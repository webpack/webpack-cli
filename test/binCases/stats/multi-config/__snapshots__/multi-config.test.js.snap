// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`multi-config 1`] = `""`;
